socket.io-client/socket.io-client.d.ts

```diff
@@ -1,46 +1,656 @@
-// Type definitions for socket.io-client 1.2.0
+// Type definitions for socket.io-client 1.3.5
 // Project: http://socket.io/
-// Definitions by: PROGRE <https://github.com/progre/>
+// Definitions by: divillysausages <https://github.com/divillysausages/>
```

Comment:
Just to clarify - we keep the original author, even if the definitions are for a different version of the lib?

I'll revert the line when I get back to my computer tonight. Tks